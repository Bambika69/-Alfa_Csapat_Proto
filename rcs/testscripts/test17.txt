createfield
createfield F1
createvirologist F1
createagent virus amnesia V1
startturn V1
use A1 V1
move F2