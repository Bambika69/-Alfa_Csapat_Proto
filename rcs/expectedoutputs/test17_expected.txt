Field created:
ID: F1
Type: Field
Neighbours: null

Field created:
ID: F2
Type: Field
Neighbours: F1

Virologist created:
ID: V1
Field: F1
Backpack: B1

Agent created:
ID: A1
Type: virus
GeneticCode: amnesia
Virologist: V1
Result: successful

V1â€™s turn started.

Agent used on Virologist:
Agent: A1
Virologist: V1
Target: V1
Result: successful

Virologist moved:
Virologist: V1
Origin: F1
Destination: F2
